core/src/main/resources/templates/error.vm

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<body>
```

Comment:
Please add `<head><title>Request handling failed</title></head>` to make the HTML valid.